flink-examples/flink-examples-streaming/pom.xml

```diff
@@ -88,6 +93,31 @@ under the License.
             <version>${project.version}</version>
         </dependency>
 
+		<!-- dependencies for MatrixVectorMul -->
+		<dependency>
+			<groupId>org.jcuda</groupId>
+			<artifactId>jcuda</artifactId>
+			<version>${jcuda.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.jcuda</groupId>
+					<artifactId>jcuda-natives</artifactId>
+				</exclusion>
```

Comment:
Please add a comment why we can exclude this dependency.

Reply:
Actually, we exclude this dependency because it is not available in all the operating systems and architectures, e.g. Jcuda 10.0.0 is not available in MacOS. We also want to enable users to compile and run the example in different runtime env. So, we provide `add-jcuda-dependency.sh` to help user to download the right native library for their runtime env.
If we do not want to include this example in `flink-dist`, I think we can just annotate the `JCuda` dependency as `provided`. WDYT?

Reply:
Thanks for the clarification. I meant an in-code comment so that other devs know why this exclusion is there.

Reply:
I also update the in-code comment as well. Not sure whether it is enough or we need more details just like my prior comment?